### DISCUSSION TITLE
new create db not show

### DISCUSSION THREAD

**Discussion by yuanhepeng** [dbeaver/dbeaver#35376] [Q&A] [answered]
### Description

I created a postgreps database in the container，mounted on volume,
![image](https://github.com/user-attachments/assets/402ca9c7-4a9d-4b9a-9f36-f942d1c5a28f)
![image](https://github.com/user-attachments/assets/9526eee4-acb2-4eb9-aa29-6b83e0c7f641)

then I created a db instance and then I restarted the container，
result db no shows, but it exists.
![image](https://github.com/user-attachments/assets/e4af2918-b083-4673-beca-c7dcb8e6986f)

In Navicat, there is no such display problem
![image](https://github.com/user-attachments/assets/51df8ae4-1d13-4964-8167-914fced9969a)

version
![image](https://github.com/user-attachments/assets/1e2de92a-8ae8-4015-a10d-d1a4a923fd69)

### DBeaver Version

24.1.5.202408181605

### Operating System

win11

### Database and driver

![image](https://github.com/user-attachments/assets/fbd07c1d-a3fe-4cce-ba0b-ecbb2b1763c7)

### Steps to reproduce

see Description

**Comment by E1izabeth** [comment, marked as answer]
Have you checked `Show all databases` button in the connection settings?
![image](https://github.com/user-attachments/assets/39266423-9ae0-46d3-94d5-cf2dd0bffc6f)

Have you pressed Refresh (F5) in navigator?

**Comment by yuanhepeng** [comment]
@E1izabeth 
Hi,thank you!
Refresh (F5) and Disconnect/Reconnect，it won't achieve the desired effect
![image](https://github.com/user-attachments/assets/2f06fe7d-baaf-4b70-8281-a31662d9c4c3)

However, 【Show all databases button in the connection settings】OK.
![image](https://github.com/user-attachments/assets/da489802-c402-4567-86da-fd357022d262)

Suggest selecting by default，because like mysql links without this setting

**Comment by E1izabeth** [comment]
Unfortunately, we can't turn it on by default. PostgreSQL's wire protocol requires opening a separate physical connection for each catalog. On top of that, for performance reasons, DBeaver opens two connections for each db instance (one general, one for metadata read). This means that if you have N catalogs in a PostgreSQL instance and 'Show all databases' on, DBeaver opens N * 2 sockets. In production environments, it's common to set an upper limit for the number of simultaneous connections from one user, and with this setting on, it's really easy to hit one.

**Reply by yuanhepeng** [reply]
Hi~,Thank you for your serious response. However, I still want to say that Navicat can turn it on by default

**Reply by yuanhepeng** [reply]
moreover， i remember my first time linking,  add one directly db is OK, able to show.
Is it possible to add a prompt to guide the user to open this setting the first time it's added
